config/initializers/0_clowder-config.rb

```diff
@@ -25,7 +25,14 @@ def build_endpoint_url(endpoint, config)
 
   # RBAC
   rbac_config = config.dependency_endpoints.dig('rbac', 'service')
-  rbac_url = build_endpoint_url(rbac_config, config)
+  rbac_hostname = rbac_config.hostname
```

Comment:
Let's name it `rbac_host` for consistency. And maybe move the port into this variable as well. See below the reasoning.